src/dfx/assets/new_project_node_files/package.json

```
@@ -2,13 +2,18 @@
   "name": "{project_name}_frontend",
   "version": "0.1.0",
   "description": "Internet Computer starter application",
-  "keywords": ["Internet Computer", "Motoko", "JavaScript", "Canister"],
+  "keywords": [
+    "Internet Computer",
+    "Motoko",
+    "JavaScript",
+    "Canister"
+  ],
   "scripts": {
     "build": "webpack",
+    "generate": "dfx generate {project_name}_backend",
     "prebuild": "npm run generate",
-    "start": "webpack serve --mode development --env development",
     "prestart": "npm run generate",
```

This may fix some of the CI errors (`dfx.json` is pointing to a `sources` npm script):

```suggestion
    "prestart": "npm run generate",
    "sources": "mops sources",
```

---

Gotcha; I see how this works now. I'll open an issue on [ZenVoich/mops](https://github.com/ZenVoich/mops) to see if this could be fixed relatively soon.

---

> Noting here - `mops sources` failed during CI with `exit status: 254`

Are there more details of the error?